### ISSUE TITLE
Heatmap tools in SEtools package

### ISSUE THREAD

**Issue by antagomir** [microbiome/OMA#127]
The SEtools package provides SE integration for pheatmap and ComplexHeatmap packages, see the [vignette](https://www.bioconductor.org/packages/release/bioc/vignettes/SEtools/inst/doc/SEtools.html).

Consider whether this would be useful to update/improve the current heatmap plotting examples in OMA. They rely on the same packages but this seems to provide added capabilities for SE objects.

**Comment by TuomasBorman** [comment]
`SEtools::sechm` is deprecated, they have moved `sechm` into [`sechm` package ](http://www.bioconductor.org/packages/release/bioc/vignettes/sechm/inst/doc/sechm.html)

I added example here
PR: https://github.com/microbiome/OMA/pull/116

I'm wondering if we should have one chapter dedicated to visualization that have all these general tricks for heatmap, boxplot, jitterplot and so on. Now these heatmap examples are under community composition.

**Comment by antagomir** [comment]
There could be a dedicated chapter on visualization, collecting key tricks in one place. 

But I would not entirely remove the examples from other chapters, either, because visualization is an important tool to understand the data also in those other chapters.

**Comment by TuomasBorman** [comment]
That's true. Currently, that heatmap example is quite long and it includes additional stuff (how to do complex hetamap with ggplot2 alone...), so that kind of tricks could have own chapter

**Comment by antagomir** [comment]
To do.

**Comment by TuomasBorman** [comment]
I think the best option is to incorporate the visualization to each topic instead of creating dedicated visualization chapter
